Packs/EclecticIQ/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,9 @@
+
+### Integrations
+#### EclecticIQ Platform
+- Command to get the Lookup Observables from the EIQ Platform
+- Command to create sighting for the EIQ data in the EIQ Platform 
+- Command to create observables for the EIQ data in the EIQ Platform 
+- Updated the Docker image to: demisto/python3:3.10.6.33415
+- Added integration support to the Playbook
+- Adpoting the package as  part of Cortex XSOAR Adopt-a-Pack program
```

Comment:
I strongly recommend using the XSOAR template for Release Notes:
General Pointers About Release Notes:
 - The release notes need to be in simple language and informative. Think about what is the impact on the user and what they should know about this version.
 - Command names - should be wrapped with three stars - ***command-name***
 - Packs/Integrations/scripts/playbooks and other content entities (incident fields, dashboards...) - should be wrapped with two stars - **entity_name**
 - Parameters/arguments/functions/outputs names - should be wrapped with one star - *entity_name*

Enhancements examples:

 - You can now filter an event by attribute data fields.

 - Added support for the *extend-context* argument in the ***ua-parse*** command.

 - Added 2 commands:
  - ***command-one***
  - ***command-two***

 - Added the **reporter** and **reporter email** labels to incidents that are created by direct messages.

 - Improved implementation of the default value for the *fetch_time* parameter.


Bug fixes examples:

 - Fixed an issue where mirrored investigations contained mismatched user names.

 - Fixed an issue with ***fetch-incidents***, which caused incident duplication.

 - Fixed an issue in which the ***qradar-delete-reference-set-value*** command failed to delete reference sets with the "\" character in their names.


Docker Updates:

 - Updated the Docker image to: *demisto/python3:3.9.1.15759*.


General Changes:
Note: Use these if the change has no visible impact on the user, but please try to refrain from using these if possible!

 - Documentation and metadata improvements.


Deprecation examples:

 - Deprecated. The playbook uses an unsupported scraping API. Use Proofpoint Protection Server v2 playbook instead.

 - Deprecated the *ipname* argument from the ***checkpoint-block-ip*** command.


Out of beta examples:

 - SaaS Security is now generally available.


The available template prefixes are:

 'Added a ',
 'Added an ',
 'Added support for',
 'Added the',
 'Completed adoption process.',
 'Deprecated the ',
 'Deprecated. ',
 'Fixed an issue',
 'Improved implementation',
 'Note: ',
 'Started adoption process.',
 'Updated the Docker image to',
 'You can now'


The available template suffixes are:

 'now generally available.'


The available full line templates are:

 'Documentation and metadata improvements.'


The BANNED line templates are:

 'maintenance and stability enhancements.',
 'stability and maintenance enhancements.'

Reply:
Updated the releasenotes as per the comment